prog/vm/github_runner.rb

```diff
@@ -125,12 +125,12 @@ def before_run
   end
 
   label def start
-    hop_wait_concurrency_limit unless concurrency_available?
+    hop_wait_concurrency_limit unless quota_available?
```

Comment:
" Add quota check for Github runner provisionings" it seems like the commit previous to this one (that adds "dark" code uncalled), pause, and then deploying "Add quota check" would be a somewhat safe deploy (& rollback) strategy.